src/FileSystem.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *  HDRMerge - HDR exposure merging software.
+ *  Copyright 2012 Javier Celaya

```

Comment:
@Hombre57 you are the author, so it should be "Copyright 2018 <Your Name> <contactinfo>"

https://www.gnu.org/licenses/gpl-howto.html
> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

http://gplv3.fsf.org/wiki/index.php/Howto